templates/html-header.twig

```diff
@@ -30,29 +30,57 @@
 	<meta name="twitter:image" content="{{ image_data[0] }}">
 
 	{% if google_tag_value %}
-		<script>
+	<script>
+          function readCookie(name) {
```

Comment:
If we put readCookie function here, we should remove it then from cookies.js .
Or we can do a quicker check with
`document.cookie.split(';').filter(function(c) {return c.indexOf('greenpeace=2') >= 0}).length`